lib/active_shipping/location.rb

```diff
@@ -13,6 +13,7 @@ class Location
       address3: [:address3],
       phone: [:phone, :phone_number],
       fax: [:fax, :fax_number],
+      email: [:email, :email_address],
```

Comment:
Do you really need `email_address` here? These attribute aliases lead to code that's hard to determine what's actually being passed in / used.